build.sbt

```diff
@@ -16,10 +16,10 @@
 
 val kamonCore       = "io.kamon" %% "kamon-core"         % "2.0.0-RC1"
 val kamonTestkit    = "io.kamon" %% "kamon-testkit"      % "2.0.0-RC1"
-val kamonScala      = "io.kamon" %% "kamon-scala-future" % "2.0.0-RC1"
-val kamonExecutors  = "io.kamon" %% "kamon-executors"    % "2.0.0-RC1"
-val kamonInstrument = "io.kamon" %% "kamon-instrumentation-common" % "2.0.0-RC1"
-val kanelaAgent     =  "io.kamon" % "kanela-agent"       % "1.0.0-M3"
+val kamonScala      = "io.kamon" %% "kamon-scala-future" % "2.0.0-RC2"
+val kamonExecutors  = "io.kamon" %% "kamon-executors"    % "2.0.0-RC2"
+val kamonInstrument = "io.kamon" %% "kamon-instrumentation-common" % "2.0.0-RC2"
+val kanelaAgent     =  "io.kamon" % "kanela-agent"       % "1.0.0-RC3"
```

Comment:
`kanela-agent` dependency should be `1.0.0-RC4`

Reply:
done